Code/GraphMol/DetermineBonds/DetermineBonds.h

```diff
@@ -0,0 +1,23 @@
+//
+//  Copyright (C) 2022 Sreya Gogineni and other RDKit contributors
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+#include <RDGeneral/export.h>
+#pragma once
+#include <GraphMol/RDKitBase.h>
+
+namespace RDKit {
+
+void determineConnectivity(RWMol &mol, bool useHuckel=false, int charge=0, double covFactor=1.3);
+
+void connectivityHuckel(RWMol &mol, int charge=0);
```

Comment:
I might remove this and 'connectivityHuckel()' from the header file next commit since they can't be called on their own (bond removal happens in 'determineConnectivity')